Update vCPU limit for UEFI guests (#458)
xcp-ng/xcp-ng-org: docs/installation/requirements.md

- **Virtual CPUs (vCPUs) per VM**:
  - For untrusted VMs, the security-supported limit is **32 vCPUs**.
  - For trusted VMs, the supported limit is **128 vCPUs** in BIOS mode and UEFI mode. Developments are planned to increase these limits.

Guest OS support is also an important factor to consider.

> > **Note**: Windows guests currently don't support beyond 64 vCPUs.

Reviewer: ```suggestion
:::note
Windows guests currently don't support beyond 64 vCPUs.
:::
```

Reviewer 2: Ping @TSnake41

- **Virtual CPUs (vCPUs) per VM**:
  - For untrusted VMs, the security-supported limit is **32 vCPUs**.
  - For trusted VMs, the supported limit is **128 vCPUs** in BIOS mode and UEFI mode. Developments are planned to increase these limits.

Guest OS support is also an important factor to consider.

:::note

> Windows guests currently don't support beyond 64 vCPUs.

Reviewer: ```suggestion
Windows guests are currently limited to a maximum of 64 vCPUs.
```

- **Virtual CPUs (vCPUs) per VM**:
  - For untrusted VMs, the security-supported limit is **32 vCPUs**.

>   - For trusted VMs, the supported limits are **128 vCPUs** in BIOS mode and UEFI mode. Developments are planned to increase these limits.

Reviewer 2: There's only one limit now, so we can go back to singular.

```suggestion
  - For trusted VMs, the supported limit is **128 vCPUs** in BIOS mode and UEFI mode. Developments are planned to increase these limits.
```
